automation/rpm.sh

```diff
@@ -28,5 +28,3 @@ find . \
     -maxdepth 1 \
     -iname 'vdsm*.tar.gz' \
     -exec mv {} "${EXPORT_DIR}/" \;
-
-createrepo_c "${EXPORT_DIR}"
```

Comment:
Can we keep this so rpm.sh can be reused locally or on gitlab?

Reply:
Done